packages/cells/src/widget.ts

```diff
@@ -1169,7 +1169,8 @@ export abstract class AttachmentsCell extends Cell {
    * Modify the cell source to include a reference to the attachment.
    */
   protected abstract updateCellSourceWithAttachment(
-    attachmentName: string
+    attachmentName: string,
+    URI: string
```

Comment:
For backwards compatibility (so we can release it in 2.1 next week), this needs to be optional.
```suggestion
    URI?: string
```